lib/recurly/hosted-field.js

```diff
@@ -82,6 +87,16 @@ export class HostedField extends Emitter {
     });
   }
 
+  bindTabTrap () {
+    this.tabTrapInput.addEventListener('focus',(e) => {
+      e.preventDefault()
+      e.stopPropagation()
+      // Firefox demands a blur or else the focus doesn' work...
+      if(typeof InstallTrigger !== 'undefined') e.currentTarget.blur()
```

Comment:
I'm a little skeptical of the single deduction. Could we do userAgent detection just as easily, with the intent being a bit more clear?

Something like `~navigator.userAgent.toLowerCase().indexOf('firefox')`